NationStatesAPIBot/NationStatesAPIBot/Services/RecruitmentService.cs

```diff
@@ -144,23 +150,36 @@ private async Task<bool> IsNationRecruitableAsync(Nation nation, EventId id)
         {
             if (nation != null)
             {
-                if (!await DoesNationFitCriteriaAsync(nation) || !await WouldReceiveTelegram(nation.Name))
+                var result = await IsNationRecruitableAsync(nation.Name, id);
+                if (!result)
                 {
-                    _logger.LogDebug(id, LogMessageBuilder.Build(id, $"{nation.Name} does not fit criteria and is therefore skipped"));
                     await NationManager.SetNationStatusToAsync(nation, "skipped");
+                }
+                return true;
```

Comment:
This returns true if a nation is skipped, no? Is that intended?

Reply:
Not it isn't.